pkg/solidity-utils/contracts/helpers/BufferHelpers.sol

```diff
@@ -20,29 +23,25 @@ library BufferHelpers {
      * - 3 underlying = 1.5 wrapped
      * - final balances: 3.5 wrapped (2 existing + 1.5 new) and 7 underlying (10 existing - 3)
      */
-    function getBufferUnderlyingSurplus(bytes32 bufferBalance, IERC4626 wrappedToken) internal view returns (uint256) {
-        uint256 underlyingBalance = bufferBalance.getBalanceRaw();
+    function getBufferUnderlyingSurplus(bytes32 bufferBalance, IERC4626 wrappedToken) internal view returns (int256) {
```

Comment:
Yes; it would be surplus / deficit. Or really, surplus / surplus in the opposite type. A negative "underlying surplus" is a wrapped surplus. Delta is a good generic term to capture the difference.